run.js

```diff
@@ -242,12 +243,47 @@ async function buildTemplates(preProcessor, postProcessor, selectedTemplate) {
                             type: m.getTemplateType()
                         }
                         templateIndex[template.getIdentifier()] = indexData;
-        
+
                         // call the post template processor
                         await postProcessor(templateIndex, templatePath, template);
                     }
                     else {
+                      if (process.env.UPDATE_DROPDOWNS) {
```

Comment:
This logic doesn't seem right. Currently, I can't `UPDATE_DROPDOWNS` and do generation in a single pass?

You could just move the `if (process.env.UPDATE_DROPDOWNS) {` block inside the `if(!process.env.SKIP_GENERATION) { ... }` block.